### PR TITLE
Refactor, style VotingPage components

### PR DESCRIPTION
**Pull request by drew-ross**
- refactor VotingForm to use buttons instead of radio inputs and render SubmissionViewer components
- move VotingFrom from common components to VotingPage directory
- add, update styles for VotingForm and other classes
- clean up, remove old code in VotingForm components
- add thumbsup.svg icon

### CODE REVIEW

**Comment by drew-ross** [comment]
Adding some style changes before requesting review/merge

File: src/components/pages/VotingPage/VotingForm.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    const child = {
+      Name: props.child.name,
+      ParentID: props.child.parentId,
+      CohortID: props.child.cohortId,
+      GradeLevel: props.child.gradeLevel,
+      VotesRemaining: props.child.VotesRemaining - 1,
+    };
+
+    updateChildData(authState, child, props.child.id);
+    postVotes(authState, body).then(res => {
```
Comment (s-munro) [inline src/components/pages/VotingPage/VotingForm.js:33]: Should we add a `catch` for error handling here?

**Review by s-munro — COMMENTED** [review]
Looks good! 👍

File: src/components/pages/VotingPage/VotingForm.js (hunk 1 above)
Comment (drew-ross) [reply src/components/pages/VotingPage/VotingForm.js:33]: Good catch. We should audit all the API calls for catch blocks.

**Review by s-munro — APPROVED** [review]

**Pull request merged**